dlp/snippets/deid/deid_fpe.go

```diff
@@ -72,8 +74,8 @@ func deidentifyFPE(w io.Writer, projectID, input string, infoTypeNames []string,
 										CryptoKey: &dlppb.CryptoKey{
 											Source: &dlppb.CryptoKey_KmsWrapped{
 												KmsWrapped: &dlppb.KmsWrappedCryptoKey{
-													WrappedKey:    keyBytes,
-													CryptoKeyName: cryptoKeyName,
+													WrappedKey:    kmsWrappedCryptoKey,
+													CryptoKeyName: keyFileName,
```

Comment:
The changes in this file are confusing for me. 
Are you moving from using file contents to kmsWrappedKey from env vars?
If yes, why is CryptoKeyName using keyFileName?

Reply:
Thank you @annadata  for your feedback, Actually, the code for the decoding key is not working properly, so the KeyFIleReading part only I have changed, also not taking it as an environment variable.

Reply:
@AarshD-crest Can you please help fix the naming that is confusing?

Reply:
@annadata The variable names in `reid_text_fpe.go` is intuitive and not confusing, But I have Updated the variable names in `deid_fpe.go` as it sounds confusing.